src/sentry/testutils/relay.py

```diff
@@ -1,40 +1,22 @@
+from typing import TYPE_CHECKING
+
 import pytest
 import requests
 import responses
 
 from sentry import eventstore
-from sentry.eventtypes import transaction
-from sentry.models import EventAttachment, Relay
+from sentry.models import EventAttachment
 from sentry.testutils.helpers import get_auth_header
 
+if TYPE_CHECKING:
+    from sentry.testutils.cases import TestCase
 
-def ensure_relay_is_registered():
```

Comment:
dead code, and also broken: accidentally uses `sentry.eventtypes.transaction` where it clearly was intended to use `django.db.transaction`